docs/modules/mapstore/pages/configuring-a-generic-maploader.adoc

```diff
@@ -70,11 +89,11 @@ instance().getConfig().addMapConfig(mapConfig);
 <2> The name of your data connection.
 
 [[mapping]]
-== SQL Mapping for the Generic MapLoader
+== SQL mapping for the generic MapLoader
 
 When you configure a map with the generic MapLoader, Hazelcast creates a xref:sql:mapping-to-jdbc.adoc[SQL mapping with the JDBC connector]. The name of the mapping is the same name as your map prefixed with `__map-store.`. This mapping is used to read data from the external system, and it is removed whenever the configured map is removed. You can also configure this SQL mapping, using <<data-connection-ref, configuration properties>>.
 
-== Configuration Properties for the Generic MapLoader
+== Configuration properties for the generic MapLoader
```

Comment:
```suggestion
== Configuration properties for the Generic MapLoader
```

Reply:
And is `generic mapStore` correct then?

Reply:
No, IMHO should have the same case. This case is used also in the title btw.

Reply:
I've looked at the other sources and I think we should refer to "generic MapLoader" in _normal_ text. I know in code it is "Generic.MapLoader" and we use that when specifically including or referring to code (if we always used code names then our docs and marketing content would be all over the place). Check latest commit for proposed version.